### PR TITLE
[ACS] add v3 deprecation warnings

### PR DESCRIPTION
**Pull request by jim-minter**
**Description**  
Add deprecation warnings for ARO v3, cc @sakthi-vetrivel

**Testing Guide**  
n/a

**History Notes**  

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by jim-minter** [comment]
Hi @arrownj CI for this change now green :-)

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -3448,6 +3448,7 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
                      no_wait=False,
                      workspace_id=None,
                      customer_admin_group_id=None):
+    logger.warning('Support for the creation of ARO 3.11 clusters ends 30 Nov 2020. Please see aka.ms/aro/4 for information on switching to ARO 4.')  # pylint: disable=line-too-long
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py:3451]: can you pls also add this in _help.py? thanks.

Comment (jim-minter) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py:3451]: @yungezz good call, done - thanks

**Review by arrownj — APPROVED** [review]
LGTM

**Pull request merged**